# snipe (Beta)

## Introduction

Snipe is a lightweight alignment-free tool that facilitates efficient assessment of sequencing experiments. Snipe is designed to handle large-scale datasets and is easily scalable to work on massive repositories like the entire Sequence Read Archive (SRA).

## Key Features
- Super lightweight and blazingly fast!
- Accurate estimation of alignment-based metrics, including sequencing depth, breadth, and mapping rate.
- Studying coverage homogeneity across different chromosomes.
- Estimating Amplicon enrichment.

## Limitations

Snipe is only tested in the mammalian space.

## Installation

Available on [Bioconda](https://anaconda.org/bioconda/snipe), [Pypi test channel](https://test.pypi.org/project/snipe/), and we are currently trying to claim the project name on [Pypi](https://github.com/pypi/support/issues/4364).

## Publication

Soon!

## Snipe Explore

Check the visualization dashboard, which allows exploring thousands of preprocessed samples from the SRA. You can also compare your experiments with them!
